### PR TITLE
Added description to README.md About custom image 

### PR DESCRIPTION
**Pull request by EladTal14**
2. By default, we'll use your GitHub photo. If you don't have a GitHub profile photo,
   or want to use a different one, add it under [public/images/people](public/images/people) in jpeg format.
   Use your GitHub username for the image, e.g. `MichalPorag.jpg`.

   Then, add `"customImage": true` to your profile JSON that you created in the previous step.

### CODE REVIEW

File: components/personCard/personCard.module.scss [hunk 1]
```diff
@@ -4,14 +4,14 @@
   width: 550px;
   height: auto;
   max-height: 220px;
-  min-height: 220px;
```
Comment (urish) [inline components/personCard/personCard.module.scss:7]: what's the reason for removing it?

Comment (EladTal14) [reply components/personCard/personCard.module.scss:7]: Did it by mistake. Although I cant see any change to the card. Is it important to set min and max to the same size?
And if it does I'll make another PR to change it back.

Comment (urish) [reply components/personCard/personCard.module.scss:7]: I'm not sure - @MichalPorag any comment on that?

also, you can make changes to the existing PR. As you long as you commit them to the branch that you used for the PR (in this case, you `master` branch), they'll be automatically added to the PR

Comment (EladTal14) [reply components/personCard/personCard.module.scss:7]: Awesome thx for the tip I'll wait for Michal's answer.

**Comment by urish** [comment]
Elad, note that the changes from #104 also went into this pull-request. In general, it's a good idea to create a different branch per pull-request, to keep the changes isolated

**Comment by urish** [comment]
Hi Elad, had a chance to look into this?

**Pull request closed without merging**